plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -1234,7 +1234,7 @@ def main():
                 module.exit_json(changed=False, vmid=vmid, msg="VM with vmid <%s> already exists" % vmid)
             elif proxmox.get_vmid(name, ignore_missing=True) and not (update or clone):
                 module.exit_json(changed=False, vmid=proxmox.get_vmid(name), msg="VM with name <%s> already exists" % name)
-            elif not (node, name):
+            elif (not node) or (not name):
```

Comment:
It might be overkill, but it could also be expressed as:
```suggestion
            elif not all(node, name):
```

Reply:
Personally I find the previous version easier to understand since I don't usually write Python.

Reply:
Suit yourself 😁